### ISSUE TITLE
Does VisionFive2 support usb microphones?

### ISSUE THREAD

**Issue by csukuangfj** [starfive-tech/VisionFive2#46]
```bash
git grep sound
```
outputs nothing in this repo.

Does VisionFive2 support USB microphones?

**Comment by easytarget** [comment]
Yes, but not with the current StarFive supplied minimal distributions. 

They have a _very_ minimally configured kernel which does not have all the necessary USB and USB-sound driver modules built by default.

You can either rebuild the kernel with these options enabled (expert) or use a mainstream distro such as Ubuntu which does come with these modules pre-compiled.

**Comment by csukuangfj** [comment]
@easytarget 

>  use a mainstream distro such as Ubuntu

Could you please post the link to the image? 

I followed the following link but it does not work.
https://doc-en.rvspace.org/VisionFive2/Quick_Start_Guide/VisionFive2_QSG/flashing_with_mac_linux.html
